week5.1-ex1.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+
+def path_fun(path):
```

Comment:
I know that paths are fun, but I didnt think you would like them that much. 
kidding aside, you dont have to add func\function to a function name, we know its a funtion